lib/screens/organization_list/organization_list_screen.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:flutter/material.dart';
+import './widgets/organization_card.dart';
+/* this is the screen which will be displayed when the user wants to see the list of organisations of which he is a part
```

Comment:
Try to use [doc comments](https://dart.dev/guides/language/effective-dart/documentation)
```suggestion
/// this is the screen which will be displayed when the user wants to see the list of organisations of which he is a part
```